.travis.yml

```diff
@@ -6,16 +6,31 @@ php:
     - 5.5
     - 5.6
     - hhvm
+    - 7.0
 
 env:
     - WP_VERSION=latest WP_MULTISITE=0
     - WP_VERSION=latest WP_MULTISITE=1
-    - WP_VERSION=4.1 WP_MULTISITE=0
-    - WP_VERSION=4.1 WP_MULTISITE=1
-    - WP_VERSION=4.0 WP_MULTISITE=0
-    - WP_VERSION=4.0 WP_MULTISITE=1
-    - WP_VERSION=3.9 WP_MULTISITE=0
-    - WP_VERSION=3.9 WP_MULTISITE=1
+    # Current stable
+    - WP_VERSION=4.5 WP_MULTISITE=0
```

Comment:
Should still be testing latest per https://github.com/Automattic/Edit-Flow/pull/336/files#diff-354f30a63fb0907d4ad57269548329e3R12, right?